node/sc/bridge_account_test.go

```diff
@@ -52,41 +54,78 @@ func TestBridgeAccountLockUnlock(t *testing.T) {
 	cPwdStr, err := ioutil.ReadFile(cPwdFilePath)
 	assert.NoError(t, err)
 
-	// Lock Account
-	{
-		err := bAcc.cAccount.LockAccount()
-		assert.NoError(t, err)
-		assert.Equal(t, false, bAcc.cAccount.IsUnlockedAccount())
-	}
-	{
-		err := bAcc.pAccount.LockAccount()
-		assert.NoError(t, err)
-		assert.Equal(t, false, bAcc.pAccount.IsUnlockedAccount())
+	lockAccountsWithCheck := func(t *testing.T, bAcc *BridgeAccounts) {
+		{
+			err := bAcc.cAccount.LockAccount()
+			assert.NoError(t, err)
+			assert.Equal(t, false, bAcc.cAccount.IsUnlockedAccount())
+		}
+		{
+			err := bAcc.pAccount.LockAccount()
+			assert.NoError(t, err)
+			assert.Equal(t, false, bAcc.pAccount.IsUnlockedAccount())
+		}
 	}
 
-	// Fail to UnLock Account
-	{
-		err := bAcc.cAccount.UnLockAccount(string(cPwdStr)[3:])
-		assert.EqualError(t, err, keystore.ErrDecrypt.Error())
+	unlockAccountsWithCheck := func(t *testing.T, bAcc *BridgeAccounts, duration *uint64, expectedErr error, expectedIsUnLock bool) {
```

Comment:
In this function, `expectedErr` and `expectedIsUnLock` are used to represent the same situation.
Since the function doesn't get a password as a parameter, the return error is only caused by invalid duration. 
Therefore, `expectedErr==nil` if `expectedIsUnlock==true`.